### PR TITLE
Fixed an issue where the focus was moving to some hidden link showing…

### PR DESCRIPTION
**Pull request by veler**
Fixed an issue where the focus was moving to some hidden link showing on left panel on documentation page.

Version for Master : [https://github.com/Microsoft/pxt/pull/2938](https://github.com/Microsoft/pxt/pull/2938)

Related issue : [https://github.com/Microsoft/pxt/issues/2795](https://github.com/Microsoft/pxt/issues/2795)

capture

I managed to support those special cases where there is no link for the header of a menu, but that the markdown parser was still generating a href tag.

capture

### CODE REVIEW

**Comment by pelikhan** [comment]
Merged in master as well?

File: docfiles/macros.html [hunk 1]
```diff
@@ -112,6 +112,14 @@
     </div>
 </aside>
 
+<aside id=top-dropdown-noHeading class=toc>
```
Comment (pelikhan) [inline docfiles/macros.html]: i think we use lower case for css classes

**Comment by veler** [comment]
@pelikhan , I will need to cherry pick to master once approved.

**Comment by microsoftsam** [comment]
@pelikhan will you apply these changes to pxt-backend once we merge?

**Comment by pelikhan** [comment]
Yes

**Comment by microsoftsam** [comment]
@veler create another PR just like this based on master, so we can merge both at the same time.

**Comment by veler** [comment]
PR for master created : [https://github.com/Microsoft/pxt/pull/2938](https://github.com/Microsoft/pxt/pull/2938)

**Pull request merged**